### PR TITLE
#160011170 Setup environment for unittests

### PR DESCRIPTION
**Pull request by petermwash**

### CODE REVIEW

**Comment by petermwash** [comment]
#### What does this PR do?
Setup environment for writing tests

#### Description of tasks to be completed
Use Jest test runner with Enzyme to setup environment for writing tests

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout -b ch-setup-environment-for-unittests-160011170
```
```sh
$ cd ah-titans-frontend
```
>Install the dependencies with the command below

```sh
$ npm install
```
>Now run the tests with the command below

```sh
$ npm test
```
>You will see the test running and passing as in the screenshot below

screen shot 2018-08-28 at 10 07 49

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160011170](https://www.pivotaltracker.com/story/show/160011170)

File: tests/app/App.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (jmnyarega) [inline tests/app/App.test.js]: Remove this line, also remember to setup your linter on your system

Comment (petermwash) [reply tests/app/App.test.js]: Okay

Comment (petermwash) [reply tests/app/App.test.js]: @jmnyarega Fixed.

**Pull request merged**